framework/docs/source/how-to-authenticate-supernodes.rst

```diff
@@ -2,90 +2,125 @@ Authenticate SuperNodes
 
-Let's break down the authentication flags:
+Enable node authentication in SuperLink
+---------------------------------------
 
-1. The first flag ``--auth-list-public-keys`` expects a path to a CSV file storing all
-   known node public keys. You need to store all known node public keys that are allowed
-   to participate in a federation in one CSV file (``.csv``).
+To launch a SuperLink with SuperNode authentication enabled, you need to provide three
+aditional files in addition to the certificates needed for the TLS connections. Recall
+that the authentication feature can only be enabled in the presence of TLS.
 
-       A valid CSV file storing known node public keys should list the keys in OpenSSH
-       format, separated by commas and without any comments. For an example, refer to
-       our code sample, which contains a CSV file with two known node public keys.
+.. code-block:: bash
+    :emphasize-lines: 5,6,7
+
+    flower-superlink \
+        --ssl-ca-certfile certificates/ca.crt \
+        --ssl-certfile certificates/server.pem \
+        --ssl-keyfile certificates/server.key \
+        --auth-list-public-keys keys/client_public_keys.csv \
+        --auth-superlink-private-key keys/server_credentials \
+        --auth-superlink-public-key keys/server_credentials.pub
 
-2. The second and third flags ``--auth-superlink-private-key`` and
-   ``--auth-superlink-public-key`` expect paths to the server's private and public keys.
-   For development purposes, you can generate a private and public key pair using
-   ``ssh-keygen -t ecdsa -b 384``.
+.. dropdown:: Understand the command
+
+    * ``--auth-list-public-keys``: Specify the path to a CSV file storing the public keys of all SuperNodes that should be allowed to connect with the SuperLink.
+      | A valid CSV file storing known node public keys should list the keys in OpenSSH format, separated by commas and without any comments. Refer to the code sample, which contains a CSV file with two known node public keys.
+    * | ``--auth-superlink-private-key``: the private key of the SuperLink.
+    * | ``--auth-superlink-public-key``: the public key of the SuperLink.
 
 .. note::
 
-    In Flower 1.9, there is no support for dynamically removing, editing, or adding
-    known node public keys to the SuperLink. To change the set of known nodes, you need
-    to shut the server down, edit the CSV file, and start the server again. Support for
-    dynamically changing the set of known nodes is on the roadmap to be released in
-    Flower 1.10 (ETA: June).
+    Currently, there is no support for dynamically removing, editing, or adding known
+    node public keys to a running SuperLink. To change the set of known nodes, you need
+    to shut the SuperLink down, edit the CSV file, and start the SuperLink again.
+    Support for dynamically changing the set of known nodes will be available in an
+    upcoming Flower release.
```

Comment:
@danielnugraha this is not true any more, right? IIRC, SuperLink re-loads the list of known public keys